docs/UserGuide.md

```diff
@@ -142,16 +145,15 @@ Examples:
 
 Deletes the specified person from the address book.
 
-Format: `delete INDEX`
+Format: `delete [P/E][INDEX]`
 
-- Deletes the person at the specified `INDEX`.
-- The index refers to the index number shown in the displayed person list.
-- The index **must be a positive integer** 1, 2, 3, …​
+* Deletes the person at the specified `INDEX` of person list (`P`) or event list (`E`).
+* The index refers to the index number shown in the respective displayed list.
+* The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-
-- `list` followed by `delete 2` deletes the 2nd person in the address book.
-- `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+* `list -p` followed by `delete 2` deletes the 2nd person in Focuris.
+* `find CS2100` followed by `delete 1` deletes the 1st Event in the results of the `find` command.
```

Comment:
For these two lines, please include P / E respectively in the indices for the delete command